packages/discovery-react-components/src/components/DocumentPreview/components/SimpleDocument/SimpleDocument.tsx

```diff
@@ -144,6 +162,12 @@ export const SimpleDocument: FC<Props> = ({
     }
   }, [passage]);
 
+  // render fallback component if specified
+  if (!html && fallbackComponent) {
+    const Fallback = fallbackComponent;
+    return <Fallback document={document} />;
```

Comment:
I think this should still be rendered within `<div className={baseClass}>`, to keep things consistent. Essentially, a "sibling" of `<ErrorView>` below.

Reply:
Thank you for the feedback. I actually thought about that for a while but couldn't get which is good. Keeping the error fallback within the `<div className>` is important to keep consistent style.